AEPMessaging/Sources/PropositionItem.swift

```diff
@@ -0,0 +1,91 @@
+/*
+ Copyright 2023 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import AEPServices
+import Foundation
+
+@objc(AEPPropositionItem)
+@objcMembers
+public class PropositionItem: NSObject, Codable {
+    /// Unique PropositionItem identifier
+    public let uniqueId: String
+
+    /// PropositionItem schema string
+    public let schema: String
+
+    /// PropositionItem data content e.g. html or plain-text string or string containing image URL, JSON string
+    public let content: String
+
+    /// Weak reference to Proposition instance
+    weak var proposition: Proposition?
```

Comment:
are we using this or is meant to be potentially used by the developer?

Reply:
Mostly by the developer.